pyuvdata/uvdata/miriad.py

```diff
@@ -775,10 +777,16 @@ def read_miriad(
             if this keyword is False. Therefore, setting read_data to False
             results in an incompletely defined object (check will not pass).
         phase_type : str, optional
-            Option to specify the phasing status of the data. Options are 'drift',
-            'phased' or None. 'drift' means the data are zenith drift data,
-            'phased' means the data are phased to a single RA/Dec. Default is None
-            meaning it will be guessed at based on the file contents.
+            Deprecated, use the `projected` parameter insted. Option to specify the
+            phasing status of the data. Options are 'drift', 'phased' or None. 'drift'
+            is the same as setting the projected parameter to False, 'phased' is the
+            same as setting the projected parameter to True. Ignored if `projected`
+            is set.
+        projected : bool or None
+            Option to force the dataset to be labelled as projected or unprojected
+            regardless of the evidence in the file. The default is None which means that
+            the projection will be set based on the file contents. Be careful setting
+            this keyword unless you are confident about the contents of the file.
```

Comment:
hmm, yes. I think in general neither keyword should be used unless you're sure. What do you suggest? We could put that warning under both keywords...

Reply:
I may just be taking this deprecation time too seriously too. Because honestly it reads fine on its own. We don't want a used with an uprojected dataset to set this keyword when it is not needed.